hopp/simulation/technologies/financial/custom_financial_model.py

```diff
@@ -393,9 +393,10 @@ def export_battery_values(self):
     def annual_energy(self) -> float:
         return self.value('annual_energy_pre_curtailment_ac')
     
-    @property
-    def om_total_expense(self) -> float:
-        return self.value('om_total_expense')
+    #below throws a recursion error in debug mode
+    # @property
+    # def om_total_expense(self) -> float:
+    #     return self.value('om_total_expense')
```

Comment:
If you replaced this with something like

`return self.Outputs. om_total_expense`

would that work?